PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+# Word Guess
+Congratulations! You're submitting your assignment.
+
+## Comprehension Questions
+
+|  Feature 	|   Feedback	|
+|---	|---	|
+|   How do you feel you and your partner did in sharing responsibilities?	|   	|
+|   Describe an instance where you used a method for something to encapsulate the functionality within your class.	|   	|
+|   Describe an instance where you used a local variable instead of an instance variable. Why did you make that choice?	|   	|
+|   What code, if any, did you feel like you were duplicating more than necessary?	|   	|
```

Comment:
Yeah... that's sort of what I was trying to get at with the local vs instance variable question. But just having them think about why since they don't always recognize right away why they shouldn't just use instance variables everywhere.